articles/quickstart/webapp/java-spring-mvc/_includes/_setup.md

```diff
@@ -31,16 +31,9 @@ com.auth0.clientId: ${account.clientId}
 com.auth0.clientSecret: YOUR_CLIENT_SECRET
 ```
 
-The library we're using has this default behavior:
-- Request the scope `openid`, needed to call the `/userinfo` endpoint later to verify the User's identity.
-- Request the `code` Response Type and later perform a Code Exchange to obtain the tokens.
-- Use the `HS256` Algorithm along with the Client Secret to verify the tokens.
-
-But it also allows us to customize its behavior:
-* To use the `RS256` Algorithm along with the Public Key obtained dynamically from the Auth0 hosted JWKs file, pass a `JwkProvider` instance to the `AuthenticationController` builder.
-* To use a different Response Type, set the desired value in the `AuthenticationController` builder. Any combination of `code token id_token` is allowed.
-* To request a different `scope`, set the desired value in the `AuthorizeUrl` received after calling `AuthenticationController#buildAuthorizeUrl()`.
-* To specify the `audience`, set the desired value in the `AuthorizeUrl` received after calling `AuthenticationController#buildAuthorizeUrl()`.
+By default, the **auth0-java-mvc-commons** library will execute the [Open ID Connect](https://openid.net/specs/openid-connect-core-1_0-final.html) Authorization Code Flow and verify the ID token using the **HS256 symmetric algorithm**. This article will demonstrate how to configure the library for use with the **RS256 asymmetric algorithm**, which is the recommended signing algorithm.
```

Comment:
I wonder if we need to go into too much back and forth about what is default and what we're doing here. Since we want folks to use RS256, maybe skip even talking about HS?

Reply:
I'll see how we can reduce discussion of HS256, while still dealing with the fact that it is the default for the mvc-commons SDK currently.

Reply:
Actually I think we can just remove this entire paragraph. I don't think it's necessary to discuss library defaults or HS or other configuration options. That information should exist in the library's documentation, and the QS can be simplified and streamlined.